boards/xtensa/intel_adsp_cavs15/tools/flash.sh

```diff
@@ -5,14 +5,16 @@
 BUILD=$1
 FIRMWARE=${BUILD}/zephyr/zephyr.ri
 FLASHER=${ZEPHYR_BASE}/boards/xtensa/intel_adsp_cavs15/tools/fw_loader.py
+CONFIGURATION=${ZEPHYR_BASE}/../modules/audio/sof/zephyr/ext/rimage/config
```

Comment:
previously somebody commented, that hard-coding module paths isn't a good idea. Not sure though whether this is a similar case too, or whether it would be acceptable here.

Reply:
Yes, maybe we can get config by command line. I will change it later.